extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -273,6 +286,8 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       'w': this.size_.width,
       'h': this.size_.height,
       'iu': slotname,
+      'npa': consentState == CONSENT_POLICY_STATE.INSUFFICIENT ||
+          consentState == CONSENT_POLICY_STATE.UNKNOWN ? 1 : null,
```

Comment:
I remember we want to distinguish between `SUFFICIENT` and  `UNKNOWN_NOT_REQUIRED`. Do we have any plan on passing the value?

Reply:
Yes, I later plan to include another ad request parameter indicating which state was used here